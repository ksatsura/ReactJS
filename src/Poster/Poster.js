//imports
import React from 'react';
import '../style.css';

const Poster = (props) => {
    
    return <img src={props.url} alt='Poster' height='400px' width='285p' />;
};

export default Poster;