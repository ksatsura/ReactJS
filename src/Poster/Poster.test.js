import React from 'react';
import Poster from './Poster.js';
import Adapter from 'enzyme-adapter-react-16';
import {shallow} from 'enzyme';

describe('Poster component', () => {

    let url;

    beforeAll(() => {
        url='../images/default.png';
    });

    it('should be render correctly', () => {
        const component = shallow(<Poster  url={url} />);
        
        expect(component).toMatchSnapshot();
    });
});
