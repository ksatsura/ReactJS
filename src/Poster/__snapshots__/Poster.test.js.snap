// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Poster component should be render correctly 1`] = `ShallowWrapper {}`;
